cebra/data/single_session.py

```diff
@@ -351,17 +359,55 @@ def __post_init__(self):
         #            e.g. integrating the FAISS dataloader back in.
         super().__post_init__()
 
-        if self.conditional != "time_delta":
-            raise NotImplementedError(
-                "Hybrid training is currently only implemented using the ``time_delta`` "
-                "continual distribution.")
-
-        self.time_distribution = cebra.distributions.TimeContrastive(
-            time_offset=self.time_offset,
-            num_samples=len(self.dataset.neural),
-            device=self.device)
-        self.behavior_distribution = cebra.distributions.TimedeltaDistribution(
-            self.dataset.continuous_index, self.time_offset, device=self.device)
+        # BEHAVIOR DISTRIBUTION
```

Comment:
auxiliary vs 'behavior'?

Reply:
I rewrote this and removed the comment.

Regarding the naming, this would be a broader change of terminology; e.g. also in non-xcebra related distributions we talk about "behavior" (vs. auxiliary). 

I opened an issue: https://github.com/AdaptiveMotorControlLab/CEBRA/issues/243